src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -587,7 +587,7 @@ public boolean isEachRoleUpdated(UserProfileUpdatedData userProfileUpdatedData,
         } catch (Exception ex) {
             log.error("{}:: UserProfile modify api failed for row ID {} with error :: {}:: Job Id {}",
                 loggingComponentName, rowId, ex.getMessage(), validationServiceFacade.getAuditJobId());
```

Comment:
make sure that this exception block is only for handling inactive user in UP, otherwise for any other failure it will always give same exception message